glotaran/project/project.py

```diff
@@ -683,7 +710,11 @@ def optimize(
         from glotaran.optimization.optimize import optimize
 
         scheme = self.create_scheme(
-            model_name, parameters_name, maximum_number_function_evaluations, clp_link_tolerance
+            model_name=model_name,
+            parameters_name=parameters_name,
+            maximum_number_function_evaluations=maximum_number_function_evaluations,
+            clp_link_tolerance=clp_link_tolerance,
+            data_lookup_overwrite=data_lookup_overwrite,
```

Comment:
```suggestion
            data_lookup_override=data_lookup_override,
```

Reply:
Fixed in [5e7a47d](https://github.com/glotaran/pyglotaran/pull/1257/commits/5e7a47de52900b22acde581c9e934280d5e95975)